src/app/ApplicationTemplate.Shared.Views/App.xaml.cs

```diff
@@ -80,6 +81,10 @@ private void InitializeAndStart()
 #endif
 //+:cnd:noEmit
 
+#if DEBUG
```

Comment:
Does this needs //-:cnd:noEmit and //+:cnd:noEmit ?

Reply:
yes